arangod/IResearch/IResearchLink.h

```diff
@@ -96,16 +96,6 @@ class IResearchLink {
   void afterTruncate(TRI_voc_tick_t tick,
                      arangodb::transaction::Methods* trx); // arangodb::Index override
 
-  ////////////////////////////////////////////////////////////////////////////////
-  /// @brief insert a set of ArangoDB documents into an iResearch View using
-  ///        '_meta' params
-  /// @note arangodb::Index override
-  ////////////////////////////////////////////////////////////////////////////////
-  virtual void batchInsert(
```

Comment:
No, they are unrelated. This just deletes the batchInsert methods that are not in use since MMFiles was removed.
Could be done in a seperate PR of course, but I think it doesn't do any harm removing them here.

Reply:
Marking this as resolved, as the deletions are intentional (though unrelated).

Reply:
Some of the `LocalTaskQueue` APIs changed, and since this code uses `LocalTaskQueue` but is completely obsolete, I figured remove it rather than update it.